src/components/form-control/form-control.test.jsx

```diff
@@ -57,6 +57,16 @@ describe('<FormControl>', () => {
       render(<FormControl {...requiredProps} error="I am an error message" />);
       expect(screen.getByText('I am an error message')).toBeInTheDocument();
     });
+
+    it('uses aria-live polite to alert users of an error', () => {
+      render(<FormControl {...requiredProps} error="I am an error message" />);
+      expect(screen.getByText('I am an error message')).toHaveAttribute('aria-live', 'polite');
+    });
+
+    it('uses a hint to alert users of the error message', () => {
+      render(<FormControl {...requiredProps} error="I am an error message" />);
```

Comment:
I think this prop switched over to `errorText`. Is it still `error` in `FormControl`? If so, we may want to update it as well.

Reply:
We can make a new story too to avoid having breaking changes in this particular PR (and thus keep the AC for the story focused)